### DISCUSSION TITLE
ads1115? Display?

### DISCUSSION THREAD

**Discussion by cjmdh** [nebhead/PiFire#162] [General]
I have an Amaz0n [ads1115](https://www.amazon.com/gp/product/B01DLHKMO2/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1) and it doesn't seem to be working since a 0 is displayed for the grill temp?  I also have the [240x320 ILI9341](https://www.amazon.com/gp/product/B01DLHKMO2/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1) display that displays the PiFire emblem during the boot sequence but that is it unless I put the controller in manual mode and even that is iffy.  Any suggestions?  I have attached logs if that is helpful.  Thanks in advance for your assistance!
 
[PiFire_Logs_09-10-23_190301.zip](https://github.com/nebhead/PiFire/files/12614961/PiFire_Logs_09-10-23_190301.zip)

**Comment by nebhead** [comment]
I don't see any errors in the control logs.  :( 

What version of PiFire are you running?  

Just to make sure, you are attempting to run the grill (i.e. startup or monitor), correct?  Because you will see zeros when the grill is inactive.

**Reply by cjmdh** [reply]
OK, I am a dork!  Didn't think to push the start button.  Now it works.  Still have a probe issue though.  Grill probe still reads 0 but probe 2 with nothing connected reads 6.  Still bench top testing so just have the grill probe and display connected.

**Reply by nebhead** [reply]
You probably need to configure your probe profiles differently.  Play around with changing the probe profiles to see if you can get some different results.

**Reply by cjmdh** [reply]
The ADS1115 is mixing up the channels.  I was reading through the exchange you had with plazarch and found that test script to output the ADS values.  Only have the grill probe connected to the board and this is the output below.  I tried to reconfigure the grill probe to use ADC1 but it still displays 0.  If I have it configured as ADC0 it displays on probe 1 spot.  I guess I will try a different ADS1115.  
One other strange thing is that unless I configure 3 probes I do not get a display when in monitor or run mode.
adc[0] = 3262.125
adc[1] = 2892.0
adc[2] = 3261.9375
adc[3] = 3261.9375
